llvm/lib/CodeGen/MachineOutliner.cpp

```diff
@@ -128,6 +135,19 @@ static cl::opt<bool> OutlinerLeafDescendants(
              "tree as candidates for outlining (if false, only leaf children "
              "are considered)"));
 
+static cl::opt<bool>
```

Comment:
These flags are currently added for testing purposes and are not strictly necessary. However, if everyone agrees, I am happy to remove them.